Assets/Scripts/Player/PlayerController.cs

```diff
@@ -191,27 +220,34 @@ public class PlayerController : MonoBehaviour {
 				BeginJump(onJump);
 				doJump = true;
 			}
-			else if (!humanControls && airJumpsUsed < airJumpsAllowed) {
+			else if (!transformation.IsHuman && airJumpsUsed < airJumpsAllowed) {
 				BeginJump(onAirJump);
 				doJump = false;
 				airJumpsUsed++;
 				doAirJump = true;
 			}
 		}
 
-		if (!humanControls && dashInputDown && allowDash)
+		if (!transformation.IsHuman && dashInputDown && allowDash)
 			BeginDash(facing);
+
+		if (!isGrounded) return;
+
+		if (transformation.IsHuman && moveInput.y < crouchInputThreshold && !isCrouched)
```

Comment:
I think it would be better to add a separate input action for crouching. In hindsight i think we should have done the same for the "up to enter" input for doors.